icaruscode/CRT/CRTDecoder/DecoderICARUSCRT_module.cc

```diff
@@ -150,8 +156,9 @@ void crt::DecoderICARUSCRT::produce(art::Event& evt)
   };
 
   // vector: Mac5 -> its CRT data
-  std::vector<icarus::crt::CRTData> allCRTdata ( 305 + 1); // TODO size this correctly!
-
+  std::vector<icarus::crt::CRTData> allCRTdata;
+  
+  std::cout << "Hit_vector.size(): " << hit_vector.size() << "\n";
```

Comment:
```suggestion
  mf::LogVerbatim("DecoderICARUSCRT") << "CRT Hit_vector.size(): " << hit_vector.size() << "\n";
```
if really necessary to see this, or otherwise `mf::LogTrace` (`DEBUG` level message) instead of `mf::LogVerbatim` (which is `INFO` level).